src/test/java/com/qa/koel/BaseTest.java

```diff
@@ -48,67 +48,23 @@ public void closeWebBrowser() {
         getDriver().quit();
     }
 
-
-
-
-
     public void createPlaylist(){
-//        WebElement plusSign = explicitWait.until(ExpectedConditions.elementToBeClickable
-//                (By.cssSelector("i[data-testid='sidebar-create-playlist-btn']")));
-//        actions.click(plusSign).perform();
+
         explicitWait.until(ExpectedConditions.visibilityOfElementLocated
                 (By.cssSelector("li[data-testid='playlist-context-menu-create-simple']")));
 
-//        WebElement optionNewPlaylist = explicitWait.until(ExpectedConditions.elementToBeClickable(By.cssSelector
-//                ("li[data-testid='playlist-context-menu-create-simple']")));
-//        actions.click(optionNewPlaylist).perform();
-
-//        WebElement inputTypeText = getDriver().findElement(By.cssSelector("form[class='create']>input"));
-//        inputTypeText.sendKeys("NewP");
-//        inputTypeText.sendKeys(Keys.ENTER);
         explicitWait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("#playlists > ul > li:nth-child(4) > a")));
     }
-    public void addSongIntoUserPlaylist() {
-//        WebElement playlistAllSongs = getDriver().findElement(By.cssSelector("a[class='songs']"));
-//        playlistAllSongs.click();
-
-//        WebElement firstSongInAllSongsList = getDriver().findElement(By.cssSelector
-//                ("#songsWrapper > div > div > div > table > tr:nth-child(1) > td.title"));
-//        firstSongInAllSongsList.click();
-
-//        WebElement buttonAddTo = getDriver().findElement(By.cssSelector("button[class='btn-add-to']"));
-//        buttonAddTo.click();
-
-//        WebElement playlistNewPinDropDownMenu = getDriver().findElement(By.cssSelector
-//                ("  #songsWrapper > header > div > div > section > ul > li.playlist"));
-//        playlistNewPinDropDownMenu.click();
-
-//        WebElement createdPlaylistNewP = getDriver().findElement(By.cssSelector
-//                ("#playlists > ul > li:nth-child(4) > a"));
-//        createdPlaylistNewP.click();
-    }
+    public void addSongIntoUserPlaylist() {}
```

Comment:
**BaseTest** — Implements the necessary methods for work with webdriver.

The BaseTest class contains common setup and teardown methods, as well as any shared functionality or configurations required by multiple test classes.

**BaseTest**

The problem we are trying to solve here is the reuse of common behaviors across test classes avoiding coding duplication and centralizing these actions at one point.

**@BeforeSuite**	- The annotated method will be run before all tests in this suite have run.

**@BeforeMethod** -	The annotated method will be run before all the test methods in the current class have been run.

**@AfterMethod**	- The annotated method will be run after each test method.

_Example without a Base Test Class_

![image](https://user-images.githubusercontent.com/104761482/233972627-2d7ffdc7-6a2f-4e9f-8a3a-a1b4a7a47bc4.png)

The methods must be stored in the `class HomePage` and another class because the elements are visible on the web site  pages

![Screenshot_1](https://github.com/Svyeta0210/A50-QA-Automation-Framework/assets/104761482/412e9902-2a0e-4475-8ffc-0bde26af9e59)